CONTRIBUTING.md

```diff
@@ -0,0 +1,63 @@
+# Contributing
+
+Hello and thank you for your interest in contributing to the 1Password `connect-helm-chart` project 👋! We sincerely appreciate the time and effort you put into improving our Helm Charts. Before you start, please take a moment to read through this guide to understand our contribution process.
+
+## Getting Started
+
+To start contributing, you'll need to fork this repository and clone it locally. Next,
+
+1. Get the latest [Helm release](https://github.com/helm/helm#install).
+2. Add the 1password repository to be able to download and install all the charts from this repository:
```

Comment:
To contribute these steps 2-4 aren't necessary, as they will be working from their local copy